components/diagnosis-suggestion/api/pom.xml

```diff
@@ -51,7 +51,7 @@
     <dependency>
       <groupId>de.charite.compbio</groupId>
       <artifactId>boqa</artifactId>
-      <version>1.0.1</version>
+      <version>1.0.2</version>
```

Comment:
Until 1.0.2 is actually released, this change is invalid. You may use `1.0-SNAPSHOT` to use the latest version.